support/cas-server-support-rest-x509/src/main/java/org/apereo/cas/support/x509/rest/config/X509RestConfiguration.java

```diff
@@ -44,10 +44,15 @@ public RestHttpRequestCredentialFactory x509RestRequestHeader() {
     
     @Override
     public void configureCredentialFactory(final ChainingRestHttpRequestCredentialFactory factory) {
-        if (x509CertificateExtractor.getIfAvailable() != null && casProperties.getRest().isHeaderAuth()) {
+        final X509CertificateExtractor extractor = x509CertificateExtractor.getIfAvailable();
+        final boolean headerAuth = casProperties.getRest().isHeaderAuth();
```

Comment:
Nitpicking tough I suspect spotbugs might consider this to be an issue: `casProperties.getRest`() can be extracted out into a single var perhaps.